src/newlines.cpp

```diff
@@ -3231,7 +3231,7 @@ void newlines_chunk_pos(c_token_t chunk_type, tokenpos_e mode)
    chunk_t *prev;
    int     nl_flag;
 
-   if ((mode & (TP_JOIN | TP_LEAD | TP_TRAIL)) == 0)
+   if ((mode & (TP_JOIN | TP_LEAD | TP_TRAIL)) == 0 && chunk_type != CT_COMMA)
```

Comment:
@mihaipopescu 
If I understand correctly what you're asking for, ```enum_comma-1.cfg``` and ```enum_comma-2.cfg``` should provide such coverage.